### PR TITLE
#HMS-127, add detailsTransferStatus to getTransfers api

### PR DESCRIPTION
**Pull request by ricky-cp**
Jira: https://currencyfair.atlassian.net/browse/HMS-127

Corresponding PR for the api updated
https://github.com/CurrencyFair/Atlas/pull/2331

### CODE REVIEW

**Comment by ricky-cp** [comment]
wrong branch

**Comment by ricky-cp** [comment]
property added (local generated result)
image

**Comment by daveearley** [comment]
@ricky-cp You can ignore the failed build. The build & deploy step only works if you create the PR from a CurrencyFair branch.

File: currencyfair.yaml [hunk 1]
```diff
@@ -6441,6 +6453,15 @@ components:
...
         - message
+    TransfersDetailsStatusObject:
+      type: object
+      properties:
+        status:
+          $ref: "#/components/schemas/TransfersDetailsStatusEnum"
+        message:
+          type: string
+          description: Status text
+          nullable: true
```
Comment (daveearley) [inline currencyfair.yaml:6468]: Could also add an example, not a blocker though.

**Review by daveearley — APPROVED** [review]

File: currencyfair.yaml (hunk 1 above)
Comment (ricky-cp) [reply currencyfair.yaml:6468]: addressed here https://github.com/CurrencyFair/xsapi/pull/15/commits/95341275eb02f166d844ac95f80fd51e0cbb8619

Comment (ricky-cp) [reply currencyfair.yaml:6468]: image

File: currencyfair.yaml [hunk 2]
```diff
@@ -5536,6 +5536,18 @@ components:
...
+    TransfersDetailsStatusEnum:
+      type: integer
+      enum:
+        - 1
+        - 2
+        - 3
+      description: |-
+        Possible values:
+        * `In progress: 1`
+        * `On hold:2`
```
Comment (garethpelly) [inline currencyfair.yaml]: Minor indentation fix for consistency
```suggestion
        * `On hold: 2`
```

Comment (ricky-cp) [reply currencyfair.yaml]: addressed here https://github.com/CurrencyFair/xsapi/pull/15/commits/3d039ec8f5a96b1e8e45e583ee12098eafe39109

**Review by garethpelly — APPROVED** [review]

**Pull request merged**